com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/executors/SqlQueryExecutor.java

```diff
@@ -0,0 +1,65 @@
+package com.sap.cloud.lm.sl.cf.persistence.executors;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import com.sap.cloud.lm.sl.cf.persistence.query.SqlQuery;
+import com.sap.cloud.lm.sl.cf.persistence.util.JdbcUtil;
+
+public class SqlQueryExecutor {
```

Comment:
After a long discussion with @enchobelezirev we came to the conclusion that this refactoring is not necessary and that the issue can be solved in a simpler way. Let's sync this in person tomorrow.